data/sql/updates/pending_db_world/rev_1603072521176787100.sql

```diff
@@ -4,6 +4,6 @@ INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1603072521176787100');
  * Update by Silker | <www.azerothcore.org> | Copyright (C)
 */
 
-UPDATE `creature_template` SET `unit_flags`=33554432, `AIName`='PassiveAI' WHERE  `entry` IN
+UPDATE `creature_template` SET `unit_flags`=`unit_flags`|33554432, `AIName`='PassiveAI' WHERE  `entry` IN
```

Comment:
Still no.
You add every flag you want to add by them self like this

unit=unit|flag1|flag2|flag3

Reply:
just add the flag you're adding, not all flags

Reply:
That's a single flag  ._.

```sql
33554432 | 0x0200 0000 | UNIT_FLAG_NOT_SELECTABLE | Can't be selected by mouse or with /target {name} command.
```

Reply:
ok